trunner/harness/unity.py

```diff
@@ -15,7 +15,8 @@ def unity_harness(dut: Dut, ctx: TestContext, result: TestResult) -> Optional[Te
     last_assertion = {}
     stats = {"FAIL": 0, "IGNORE": 0, "PASS": 0}
     results = []
-    timeout_val = 30
+    # some unity tests (e.g. mprotect) take 20 or even 30+ seconds on zynqmp-qemu
```

Comment:
Also the explanation needed in the commit description, sth like:

`The QEMU target for this architecture is slower than for the other ones; that's why it needs to be treated individually.`